### ISSUE TITLE
node-zopfli build process requires g++, not mentioned in README.

### ISSUE THREAD

**Issue by jsnshrmn** [edgi-govdata-archiving/web-monitoring-ui#236]
## Expected Behavior
yarn install should be happy in dev

## Current Behavior
yarn install fails, because the [required binary release of node-zopfli](https://node-zopfli.s3.amazona.ws.com/Release/zopfli-v2.0.2-node-v57-linux-x64.tar.gz) is no longer available. It's an outstanding issue with the [zopfli-webpack-plugin](https://github.com/webpack-contrib/zopfli-webpack-plugin/issues/13), which seems to be a bit under maintained itself. Additionally fallback builds are failing for me ~~and I'm not sure why~~ because the make configuration used by the build script doesn't support the c++ extensions for gcc. g++ is required.

## Possible Solutions
* add a note about the g++ dependency for building.
* and a compiled artifact as a resource for the build scripts for this project.
* you could eliminate the dependency upon zopfli-webpack-plugin, I have no idea how much work that would be. ~~I'm naively attempting that in PR https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/237~~

## Steps to Reproduce (for bugs)
```source ~/py2_env/bin/activate && source ~/.nvm/nvm.sh && yarn install --no-cache --frozen-lockfile```

## Context
I'm trying to run through local installation of the various web-monitoring bits while I'm still ignorant of the underlying code. This means that I should be encountering the same kinds of issues as potential new contributors.

## Your Environment
* This was run from ansible.
* centos 7.x
* nvm 0.33.6
* npm 3.10.10
* node 8.10

**Comment by jsnshrmn** [comment]
Continuing to poke around, I can see that node-gyp is falling back to a build from source, but that it's failing.  I'll see if I can track that down any further.

**Comment by jsnshrmn** [comment]
Here's some relevant build log
```
11528 verbose stack Error: node-zopfli@2.0.2 install: `node-pre-gyp install --fallback-to-build`
11528 verbose stack Exit status 1
11528 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
11528 verbose stack     at emitTwo (events.js:106:13)
11528 verbose stack     at EventEmitter.emit (events.js:191:7)
11528 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/lib/utils/spawn.js:40:14)
11528 verbose stack     at emitTwo (events.js:106:13)
11528 verbose stack     at ChildProcess.emit (events.js:191:7)
11528 verbose stack     at maybeClose (internal/child_process.js:920:16)
11528 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:230:5)
11529 verbose pkgid node-zopfli@2.0.2
11530 verbose cwd /home/vagrant
11531 error Linux 3.10.0-693.21.1.el7.x86_64
11532 error argv "/usr/bin/node" "/bin/npm" "install" "-g" "node-zopfli"
11533 error node v6.14.3
11534 error npm  v3.10.10
11535 error code ELIFECYCLE
11536 error node-zopfli@2.0.2 install: `node-pre-gyp install --fallback-to-build`
11536 error Exit status 1
11537 error Failed at the node-zopfli@2.0.2 install script 'node-pre-gyp install --fallback-to-build'.
11537 error Make sure you have the latest version of node.js and npm installed.
11537 error If you do, this is most likely a problem with the node-zopfli package,
11537 error not with npm itself.
11537 error Tell the author that this fails on your system:
11537 error     node-pre-gyp install --fallback-to-build
11537 error You can get information on how to open an issue for this project with:
11537 error     npm bugs node-zopfli
11537 error Or if that isn't available, you can get their info via:
11537 error     npm owner ls node-zopfli
11537 error There is likely additional logging output above.
11538 verbose exit [ 1, true ]
```

**Comment by jsnshrmn** [comment]
This is looking like a compiler dependency issue. More to come.

**Comment by jsnshrmn** [comment]
The build requires gcc-c++ where I had only gcc installed. None of the stack messages indicated that I didn't have the correct compiler! Might be worth noting in the zopfli section in the readme, as many rhel folks do many builds with only gcc installed.

**Comment by jsnshrmn** [comment]
Closed by pr #243
